airflow/cli/commands/kubernetes_command.py

```diff
@@ -91,9 +91,24 @@ def cleanup_pods(args):
     kube_client = get_kube_client()
     print(f'Listing pods in namespace {namespace}')
     list_kwargs = {"namespace": namespace, "limit": 500}
+    airflow_pod_labels = [
+        'airflow-worker',
+        'dag_id',
+        'task_id',
+        'execution_date',
+        'try_number',
+        'airflow_version',
+        'kubernetes_executor',
+    ]
     while True:  # pylint: disable=too-many-nested-blocks
         pod_list = kube_client.list_namespaced_pod(**list_kwargs)
```

Comment:
Can we use `label_selector` parameter to do server-side filtering?

Reply:
Definitely agree. Many thanks for the suggestion!

Changes pushed.